transcribe.py

```diff
@@ -104,11 +104,11 @@ def send_transcript(data, host: str = private_secrets.databaseIPV4, port: int =
                     start_time = datetime.datetime.strptime(r.json()['data'][0]['started_at'], '%Y-%m-%dT%H:%M:%SZ')
                     # Offset start time to local timezone
                     start_time = start_time + datetime.timedelta(hours=-6)  # change this to your UTC offset (negative = western hemisphere)
-                    streams[key]['start_time'] = start_time
+                    streams['start_time'] = start_time
```

Comment:
Recurring issue, I think this should become `stream["dict_item"]` instead of `streams[]`